templates/item/weapon-sheet.html

```diff
@@ -100,7 +100,7 @@ <h1 class="charname"><input name="name" type="text" value="{{name}}" placeholder
                 {{#if system.explosive}}
                 <div class="resource numeric-input flexrow">
                     <label class="resource-label">{{localize "YZECORIOLIS.BlastPower"}}</label>
-                    <input type="number" min="0" name="system.crit.blastPower" value="{{system.crit.blastPower}}"
+                    <input type="number" min="0" name="system.blastPower" value="{{system.blastPower}}"
```

Comment:
Goo catch!

I was wondering, I think it would be better to change the order of MELEE and EXPLOSIVE checkboxes in the sheet, so when you check EXPLOSIVE, the two new fields appear juste below.

Also, the term in the book is also Blast Power so maybe we could change the text as well.

<img width="715" alt="Screen Shot 2023-06-14 at 3 15 05 PM" src="https://github.com/winks-vtt/yze-coriolis/assets/408380/2c805df9-8ce7-4fe0-a93d-6ec939666a7f">